internal/models/reservation_model.go

```diff
@@ -44,23 +68,14 @@ type AWSReservation struct {
 	// Source ID.
 	SourceID string `db:"source_id" json:"source_id"`
 
-	// Optional instance name
-	Name string `db:"name" json:"name"`
-
-	// AWS Instance type.
-	InstanceType string `db:"instance_type" json:"instance_type"`
-
-	// Amount of instances to provision of type: Instance type.
-	Amount int32 `db:"amount" json:"amount"`
-
-	// The ID of the image from which the instance is created.
-	ImageID string `db:"image_id" json:"image_id"`
-
-	// Immediately power off the system after initialization
-	PowerOff bool `db:"poweroff" json:"poweroff"`
-
 	// The ID of the aws reservation which was created.
 	AWSReservationID string `db:"aws_reservation_id" json:"aws_reservation_id"`
+
+	// The ID of the image from which the instance is created. AMI's must be prefixed with 'ami-'.
+	ImageID string `json:"image_id"`
+
+	// Detail information is stored as JSON in DB
+	Detail *AWSDetail `db:"detail" json:"detail"`
```

Comment:
Here is where it get's referenced, `pgx` driver works seamlessly for this.

I kept `ImageID` here because it is a good idea as we will have to search by image id at some point.